### ISSUE TITLE
Clarify why you might have "no matching events" in an insight

### ISSUE THREAD

**Issue by corywatilo** [PostHog/posthog#23168]
_This is sort of a "works as expected" thing, but it's very counterintuitive and the UX is confusing as-is._

I was having trouble getting event properties to populate.

<img width="742" alt="image" src="https://github.com/PostHog/posthog/assets/154479/acb98aff-7f8b-4599-acbe-544bf5a3af8a">

I knew the data was there, but I it wouldn't populate in the autocomplete popover. Then I realized the properties that loads in the dropdown are time-bound based on the date range set in the results area.

In other words, no events matched in the last 7 days, but changing the time range to last 180 days then made it possible to see data in the dropdown.

I realize we're probably unlikely to change anything here, but as a user, I'd expect the dropdown to show any matching data from all time, _not_ bound to the date range set in a different section. (I didn't actually realize the data that loads in the query builder was bound to the time range set in the results area until just now.)
